<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="sellers">
         <attribute name="title">
          <string>Sellers</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QTableView" name="sellers_table_view">
                <attribute name="horizontalHeaderVisible">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderCascadingSectionResizes">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderHighlightSections">
                 <bool>true</bool>
                </attribute>
                <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                 <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                 <bool>true</bool>
                </attribute>
                <attribute name="verticalHeaderHighlightSections">
                 <bool>true</bool>
                </attribute>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLineEdit" name="name_edit">
                  <property name="placeholderText">
                   <string>Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="surname_edit">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="placeholderText">
                   <string>Surname</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="patronymic_edit">
                  <property name="placeholderText">
                   <string>Patronymic</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QPushButton" name="add_seller">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove_seller">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="products">
         <attribute name="title">
          <string>Products</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTableView" name="products_table_view">
              <attribute name="horizontalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="type_edit">
                <property name="placeholderText">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="price_edit">
                <property name="placeholderText">
                 <string>Price</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="quantity_edit">
                <property name="placeholderText">
                 <string>Quantity</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="seller_id_combobox">
                <property name="placeholderText">
                 <string>Seller Id</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="add_product">
                <property name="text">
                 <string>Add</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="remove_product">
                <property name="text">
                 <string>Remove</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="accounting">
         <attribute name="title">
          <string>Accounting</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QComboBox" name="query_select">
                <item>
                 <property name="text">
                  <string>ФИО по типу товара</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Выручка</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="type_select">
                <property name="placeholderText">
                 <string>Type</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="execute_query_btn">
                <property name="text">
                 <string>Execute query</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableWidget" name="accounting_table"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="connection_lbl">
        <property name="text">
         <string>Connection state: disconnected.</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="connect_btn">
          <property name="text">
           <string>Connect to DB</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
